test/integration/features/provision_various_versions.feature

```diff
@@ -1,4 +1,4 @@
-@provision-various-versions
+@provision-various-versions @openshift @minishift-only
```

Comment:
@amitkrout , could you remove `@minishift-only` here? We can use same testcases with different table in downstream.

Reply:
@budhrg https://github.com/minishift/minishift/pull/2044/files#r170179932

Reply:
@budhrg removed the ```@minishift-only``` tag